corehq/apps/export/views.py

```diff
@@ -2283,6 +2286,56 @@ def get_filters(self, filter_form_data, mobile_user_and_group_slugs):
         return form_filters
 
 
+class DownloadNewSmsExportView(GenericDownloadNewExportMixin, BaseDownloadExportView):
```

Comment:
Since this is a "static" report I didn't make those. When you click "SMS Exports" you're automatically taken to the download page because there's nothing to edit or create